### PR TITLE
Changelog for v0.24.0 release

### PR DESCRIPTION
**Pull request by dhavalshreyas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
 
+## Version 0.24.0
+
+_2020-03-09_
+
+### Kotlin
+
+ * Introduce a view binding function that works with AndroidX ViewBindings. #985
+ * Rename ViewBinding to ViewFactory. #1009
+ * Rename ContainerHints to ViewEnvironment to match Swift. #1005
```
Comment (zach-klippenstein) [inline CHANGELOG.md]:
```suggestion
 * Rename ContainerHints to ViewEnvironment to match Swift. #1005
 * Fix type parameter for expectWorker accepting Worker instance. (#969)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+### Swift
+
+ * Removes ViewRegistry, Introduce ViewControllerDescription. #974
+ * Introduce ViewEnvironment. #999
```
Comment (zach-klippenstein) [inline CHANGELOG.md]: Nit: For consistency with the other entries in this file, wrap the issue numbers in parentheses.

```suggestion
 * Introduce a view binding function that works with AndroidX ViewBindings. (#985)
 * Rename ViewBinding to ViewFactory. (#1009)
 * Rename ContainerHints to ViewEnvironment to match Swift. (#1005)

### Swift

 * Removes ViewRegistry, Introduce ViewControllerDescription. (#974)
 * Introduce ViewEnvironment. (#999)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
 Change Log
 ==========
 
+## Version 0.24.0
+
+_2020-03-09_
```
Comment (zach-klippenstein) [inline CHANGELOG.md]: Nit: Should probably be -03-10 at this point.

Comment (dhavalshreyas) [inline CHANGELOG.md]:
```suggestion
_2020-03-10_
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
 
+## Version 0.24.0
+
+_2020-03-10_
+
+### Kotlin
+
+ * Introduce a view binding function that works with AndroidX ViewBindings. (#985)
+ * Rename ViewBinding to ViewFactory. (#1009)
+ * Rename ContainerHints to ViewEnvironment to match Swift. (#1005)
```
Comment (dhavalshreyas) [inline CHANGELOG.md]:
```suggestion
 * Rename ContainerHints to ViewEnvironment to match Swift. (#1005)
 * Fix type parameter for expectWorker accepting Worker instance. (#969)
```

**Pull request merged**